handlers/admin_handler.py

```diff
@@ -6,6 +7,67 @@
 from util.util import (get_display_name_from_user, parse_user_id,
                        resolve_user_by_user_id)
 
+class MakeCTFCommand():
+    """
+        Update the channel purpose to be that of a CTF.
+    """
+    @classmethod
+    def execute(cls, slack_wrapper, args, timestamp, channel_id, user_id, user_is_admin):
+        if user_is_admin:
+            purpose = {}
+            purpose["ota_bot"] = "OTABOT"
+            purpose["name"] = args[0]
+            purpose["type"] = "CTF"
+            purpose["cred_user"] = ""
+            purpose["cred_pw"] = ""
+            purpose["long_name"] = args[0]
+            purpose["finished"] = False
+            purpose["finished_on"] = ""
+            slack_wrapper.set_purpose(channel_id, purpose)
+
```

Comment:
This command only sets the purpose of the channel, but is not adding it to the ctf list in the "db". Wouldn't this require a !reload after the purpose of the ctf channel is set?